test/primitives/base-element-test.js

```diff
@@ -231,4 +231,158 @@ describe('BaseElement', function() {
       });
     });
   });
+
+  context('when querying', function() {
+    it('returns null when there are no matching elements', function() {
+      const element = new minim.BaseElement();
+      const result = element.findRecursive('string');
+
+      expect(result).to.equal(null);
```

Comment:
since this should return something like array, wouldn't it make sense to return an empty array in this case?